inference-engine/tests/ngraph_helpers/ngraph_functions/include/ngraph_functions/builders.hpp

```diff
@@ -139,6 +139,20 @@ std::shared_ptr<ngraph::Node> makeConvolutionBackpropData(const ngraph::Output<N
                                                           bool addBiases = false,
                                                           const std::vector<float> &biasesWeights = {});
 
+std::shared_ptr<ngraph::Node> makeConvolutionBackpropData(const ngraph::Output<Node> &in,
```

Comment:
As I see we have 3 different builders for deconvolution. Do we need all these builders?

Reply:
Yes, we need another builder. The 2 existing builders are responsible for creating ConvolutionBackprop for this ctor [ctor](https://github.com/openvinotoolkit/openvino/blob/4bbbf66a69b339e27a22c18d9b2a485a22a77226/ngraph/core/include/ngraph/op/convolution.hpp#L137), and we need also builder for 2nd [ctor](https://github.com/openvinotoolkit/openvino/blob/4bbbf66a69b339e27a22c18d9b2a485a22a77226/ngraph/core/include/ngraph/op/convolution.hpp#L111) as it has 3rd input: output_shape.